server.go

```diff
@@ -420,6 +420,23 @@ func (s *ServerImpl) SetActiveById(id string) error {
 	return fmt.Errorf("No deploy %s, run 'list' to see valid deploys", id)
 }
 
+func (s *ServerImpl) GetFullDeployIdFromShortName(deployShortName string) (string, error) {
+  var matchingIds []string
+  for _, deployId := range s.readDeployIdsFromDisk() {
+    shortenedId := strings.TrimPrefix(deployId, deployShortName)
```

Comment:
I think using HasPrefix would be clearer
https://golang.org/pkg/strings/#HasPrefix